dbt_platform_helper/providers/io.py

```diff
@@ -7,10 +7,16 @@ class ClickIOProvider:
     def warn(self, message: str):
         click.secho(message, fg="magenta")
 
+    def debug(self, message: str):
+        click.secho(message, fg="green")
+
     def error(self, message: str):
         click.secho(f"Error: {message}", fg="red")
 
-    def info(self, message: str):
+    def info(
+        self,
+        message: str,
+    ):
```

Comment:
Seems odd that this one is multi-lined and the others are not...
```suggestion
    def info(self, message: str):
```

Reply:
I'll see if the linter picks this up